deployments/optimism/Optimism_SpokePool.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0xa420b2d1c0841415A695b81E5B867BCD07Dff8C9",
+  "address": "0x6f26Bf09B1C792e3228e5467807a900A503c0281",
```

Comment:
Same, updated this so we don't accidentally send transactions to the old address.